### PR TITLE
update package to use mm 6.3.0

### PR DESCRIPTION
**Pull request by spirosoik**
#### Summary
update package to use mm 6.3.0

#### Ticket Link
Ticket: https://mattermost.atlassian.net/browse/DOPS-763

### CODE REVIEW

**Review by amyblais — APPROVED** [review]

File: version.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 variables:
   MAJOR: 6
-  MINOR: 2
-  PATCH: 1
+  MINOR: 3
+  PATCH: 0
```
Comment (amyblais) [inline version.yml:5]:
```suggestion
  PATCH: 1
```

Should this be "1" (based on https://mattermost.atlassian.net/browse/DOPS-776)?

**Review by amyblais — CHANGES_REQUESTED** [review]

File: version.yml (hunk 1 above)
Comment (amyblais) [reply version.yml:5]: Ok, sounds like it needs a separate PR.

**Review by amyblais — APPROVED** [review]

File: version.yml (hunk 1 above)
Comment (spirosoik) [reply version.yml:5]: yeah that was my intention, separate PR for 6.3.1

Comment (cpanato) [reply version.yml:5]: yes, we should release 6.3.0 and then 6.3.1

**Review by cpanato — APPROVED** [review]
after the merge need to create the branch and then the tag, note: follow the tag pattern

**Pull request merged**